webservices/resources/reports.py

```diff
@@ -1,592 +1,37 @@
-    'items_on_hand_liquidated': fields.Integer,
-    'loans_received_from_candidate_period': fields.Integer,
-    'loans_received_from_candidate_ytd': fields.Integer,
-    'net_contribution_summary_period': fields.Integer,
-    'net_operating_expenses_summary_period': fields.Integer,
-    'offsets_to_fundraising_exp_ytd': fields.Integer,
-    'offsets_to_fundraising_expenses_period': fields.Integer,
-    'offsets_to_legal_accounting_period': fields.Integer,
-    'offsets_to_legal_accounting_ytd': fields.Integer,
-    'offsets_to_operating_expenditures_period': fields.Integer,
-    'offsets_to_operating_expenditures_ytd': fields.Integer,
-    'operating_expenditures_period': fields.Integer,
-    'operating_expenditures_ytd': fields.Integer,
-    'other_loans_received_period': fields.Integer,
-    'other_loans_received_ytd': fields.Integer,
-    'other_receipts_period': fields.Integer,
-    'other_receipts_ytd': fields.Integer,
-    'refunded_individual_contributions_ytd': fields.Integer,
-    'refunded_other_political_committee_contributions_period': fields.Integer,
-    'refunded_other_political_committee_contributions_ytd': fields.Integer,
-    'refunded_political_party_committee_contributions_period': fields.Integer,
-    'refunded_political_party_committee_contributions_ytd': fields.Integer,
-    'refunds_individual_contributions_period': fields.Integer,
-    'repayments_loans_made_by_candidate_period': fields.Integer,
-    'repayments_loans_made_candidate_ytd': fields.Integer,
-    'repayments_other_loans_period': fields.Integer,
-    'repayments_other_loans_ytd': fields.Integer,
-    'subtotal_summary_period': fields.Integer,
-    'total_disbursements_summary_period': fields.Integer,
-    'total_loan_repayments_made_period': fields.Integer,
-    'total_loan_repayments_made_ytd': fields.Integer,
-    'total_loans_received_period': fields.Integer,
-    'total_loans_received_ytd': fields.Integer,
-    'total_offsets_to_operating_expenditures_period': fields.Integer,
-    'total_offsets_to_operating_expenditures_ytd': fields.Integer,
-    'total_period': fields.Integer,
-    'total_receipts_summary_period': fields.Integer,
-    'total_ytd': fields.Integer,
-    'transfer_from_affiliated_committee_period': fields.Integer,
-    'transfer_from_affiliated_committee_ytd': fields.Integer,
-    'transfer_to_other_authorized_committee_period': fields.Integer,
-    'transfer_to_other_authorized_committee_ytd': fields.Integer,
-}
-pagination_fields = {
-    'per_page': fields.Integer,
-    'page': fields.Integer,
-    'count': fields.Integer,
-    'pages': fields.Integer,
+
+reports_schema_map = {
+    'P': (models.CommitteeReportsPresidential, schemas.ReportsPresidentialPageSchema),
+    'H': (models.CommitteeReportsHouseOrSenate, schemas.ReportsHouseSenatePageSchema),
+    'S': (models.CommitteeReportsHouseOrSenate, schemas.ReportsHouseSenatePageSchema),
+    None: (models.CommitteeReportsPacOrParty, schemas.ReportsPacPartyPageSchema),
```

Comment:
Do we want to be more precise here? PACs and Parties are committee types: N, O, Q, V, W, X, Y and Z.

We currently don't have types- I, C, D, U because their information is not in these tables. These tables are populated by the three versions of form 3.

And if we do want to be more precise, I can confirm the type-codes with Paul or Jeff.